Mechanisms_evolution/Mechanisms_Evolution.Rmd

```diff
@@ -9,252 +9,222 @@ output:
+**Make a new project and open a new script.**
 
-For this lab we will provide you with the directions of what to do by showing you the comments you should enter in you R script.
+Begin by loading all the libraries you need. In this case, we will be using `ggplot2` to make our graphs. We are also entering data into Google Sheets, so we will need `gsheet` to pull that data into R. 
 
 * Copy the comment lines into your script
-* Enter the necessary code based on the previous lab. In any case where you have not seen the necessary code it is provided. **In cases where you should know the code already you should check assignment 1 if you need help.**
+* Enter the necessary code based on the previous lab and your R script from Lab 1. **In cases where you should know the code already you should check the Lab 1 Manual if you need help.**
 
 ```{r load1}
-# Load the libraries ggplot2 and gsheet
+# Load the packages ggplot2, gsheet, and dplyr
 ```
 
 ```{r load2, echo=FALSE}
 library(ggplot2)
 library(gsheet)
+library(dplyr)
 ```
 
-## 2. Get Data into R
-
-We will use a similar procedure as the last lab to load data into R so that we can work with it.
+## 2. Get your data into R
 
-Next, load in the data (i.e. tell R where your dataset is). Today's data is found at:
-
-https://docs.google.com/spreadsheets/d/1t63u8_5onAk4PNK1bATVSGVxwOcMzvPLuIInP1UgBzE
+We will use the same procedure as the last lab to load data into R.
+Go to the Lab Sakai Site and scroll down like you did last week. Select the link for your lab section.
 
 ```{r load3}
-# Assign website address for the data to a variable
+# Store the Google sheets link as the variable url
 ```
 
 ```{r load4, echo=FALSE}
 url <- 'https://docs.google.com/spreadsheets/d/1t63u8_5onAk4PNK1bATVSGVxwOcMzvPLuIInP1UgBzE'
 ```
 
 ```{r }
-# Load the data from google sheets and store in a variable
+# load the data stored in the variable url
+# use the gsheet2tbl function
+# store it as the variable snail_data
 ```
 
 ```{r read, echo=FALSE}
 snail_data <- gsheet2tbl(url)
 ```
 
-Once we have our data in, it is always good to check to make sure the data was imported correctly. To view ONLY the first few lines of the dataset use the command `head`.
-
-```{r head, eval=FALSE}
-# Check your data looks right (first lines only)
-head(snail_data)
-```
-## 3. Find The Average Snail Count in the Oystercatcher Data
-
-We are interested in differences in population size between red and white snails over time. To graph the average snail population size over time, you first need to calculate the class averages for each snail type at each generation time for each experiment. Start with the oystercatcher data (experiment 1). 
+Once we have our variables, it is always good to check to make sure the data was imported correctly. Click on the variables listed in the R Environment (top right) and compare them to the Google Sheet data. 
```

Comment:
Needs a compromise here - Once we have our data stored in a variable...
It's worth checking by clicking but also put back `head()` - they need to know how to view the head of a really large dataset that shouldn't be clicked and viewed entirely